metadata.json

```
@@ -10,13 +10,18 @@
     {
       "name": "puppetlabs-stdlib",
       "version_requirement": ">= 4.25.1 <5.0.0"
```

Is it also worth bumping this version while we're updating things?

---

Yeah, that's probably a good idea.
```suggestion
      "version_requirement": ">= 4.25.1 <5.1.0"
```
There are no breaking changes between 5.0.0 and 5.1.0 so that's an easy bump.